code/src/components/ButtonNext.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+const ButtonNext = ({ questionIndex, setQuestionIndex }) => {
+
+    const onQuestionIndexChange = () => {
+        console.log(questionIndex)
+        setQuestionIndex(questionIndex +1)
+    }
+
+    return (
+        <>
+            <button 
+                type="button"
```

Comment:
You could set type="submit" instead and then it's directly connected with the onSubmit in the form